libs/gltfio/src/AssetLoader.cpp

```diff
@@ -156,6 +156,9 @@ struct FAssetLoader : public AssetLoader {
     const char* mDefaultNodeName;
     bool mError = false;
     bool mDiagnosticsEnabled = false;
+
+    // Weak reference to the largest dummy buffer so far in the current loading phase.
+    BufferObject* mDummyBufferObject;
```

Comment:
Aren't you leaking the object? I see no destruction anywhere.

Reply:
It is owned by the asset, in its mBufferObjects array.